database/migrations/2016_08_18_152505_create_members_table.php

```diff
@@ -12,17 +12,15 @@ class CreateUsersTable extends Migration
      */
     public function up()
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('members', function (Blueprint $table) {
```

Comment:
Existing migrations should NEVER be modified since they will break existing installations